<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetUnlockPattern</class>
 <widget class="QWidget" name="SetUnlockPattern">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label_title">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Set Unlock Pattern</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_background" native="true">
     <property name="styleSheet">
      <string notr="true">#widget_background
{
	top:-15px;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="topMargin">
          <number>20</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>25</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="GustureLock" name="widget_pattern" native="true">
           <property name="minimumSize">
            <size>
             <width>240</width>
             <height>240</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>240</width>
             <height>240</height>
            </size>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhPreferNumbers</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>37</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_patternTips">
        <property name="styleSheet">
         <string notr="true">#label_patternTips
{
color:#4a4a4a;
background-color: #FFFFFF;
}</string>
        </property>
        <property name="text">
         <string>Please draw at least 4 points.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="MyPushButton" name="pushButton_ok">
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyPushButton" name="pushButton_cancel">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyPushButton</class>
   <extends>QPushButton</extends>
   <header>mypushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>GustureLock</class>
   <extends>QWidget</extends>
   <header>gusturelock.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
